docs/pages/Cardinal/API-Reference/Query.mdx

```diff
@@ -3,12 +3,12 @@ import { Callout } from 'nextra/components'
 
 import Link from 'next/link'
 
-# Reads
+# Queries
 
-Reads enable Cardinal to handle requests for data from game state.
+Queries enable Cardinal to handle requests for data from game state.
```

Comment:
Is it worth adding some distributed system related context about how queries will return live data and are non-blocking, as opposed to how txs work? I feel that I would want to know this as a user and would hate to have to read the code to find out. 

I'd be fine if you don't wanna include this suggestion but I really don't like when docs are catered to the lowest common denominator, we must encourage the LCD to upgrade 🧠

Reply:
I added a "Query Data Guarantees" section to this file.